openapi/multi-yaml/index.yaml

```diff
@@ -79,6 +79,223 @@ tags:
       | rejected           | this sub account didn't pass approval, so they won't be able to process payments |
       | disabled           | this sub account was previously approved, but has since become ineligible to process payments (e.g. due to fraud) |
       | archived           | this sub account has been archived; they won't be able to process payments (but their record will remain for historical reasons) |
+  - name: Platform Wallet Accounts
+    description: |
+      Platform wallet accounts allow you to store payment methods centrally and use them across multiple sub-accounts within your platform. 
+      This feature enables platforms to maintain a single source of stored payment methods while processing payments through different sub-accounts.
+
+      ### Enabling Platform Wallet Accounts
+      Contact JustiFi support to enable the `platform_wallet_account` setting for your designated wallet account.
+      Once enabled, you can:
+      - Store payment methods in the designated wallet account
```

Comment:
For consistency I'd prefer if we changed `wallet account` to `platform wallet account` everywhere (despite the length of it). Would you mind changing it?